### ISSUE TITLE
Result stucks in pending

### ISSUE THREAD

**Issue by Craftoncu** [bdr99/node-mymazda#4]
```
import MyMazda from \"node-mymazda\";

async function test() {
    // Initialize API Client (MNAO = North America)
    let client = new MyMazda(\"censored\", \"censored\", \"MME\");

    // Get list of vehicles from the API (returns a list)
    console.log(client.getVehicles());
}
test();
```

Hey! With following code, the Promise (Promise<Vehicle[]>) stucks in pending.
The python lib is currently working. Bot badly this not :(

**Comment by bdr99** [comment]
Hi @Craftoncu, thanks for bringing this up.

Yes, Mazda has recently made some changes to their API, and therefore some changes are needed to this library. I have made those changes to the Python version but not to this version yet. I hope to have some time to work on this soon.

**Comment by Craftoncu** [comment]
Hey @bdr99 Brandon,

I'm little bit in a hurry to get this library working again. If you like to, tell me what have been changed and I'm used to get it working again. Its a little bit difficult to read the changes from your last commits on the python version.

**Comment by bdr99** [comment]
I actually had already completed most of the changes, so I just went ahead and finished the changes and published a new version to npm (1.1.0). Can you please try version 1.1.0 and let me know if it works for you?

**Comment by Craftoncu** [comment]
Big thanks! So far I haven't noticed anything.

**Comment by bdr99** [comment]
Hi @Craftoncu is everything still working well? I'd like to close this issue but wanted to check first to make sure you weren't seeing any issues with version 1.1.0.

**Comment by Craftoncu** [comment]
Hey @bdr99 I think Mazda made some breaking changes to their api. My adapter for iobroker doesnt work anymore...

**Comment by bdr99** [comment]
Hi @Craftoncu, yes, Mazda recently made a change to their API. I have just published version 1.1.1 which should fix the issue. Can you test it and let me know if it resolves your issue?

**Comment by bdr99** [comment]
@Craftoncu I'm closing this issue as I believe the problem is resolved. If you are still having problems then please feel free to open a new issue.
